README.md

```diff
@@ -39,3 +39,9 @@ calls to require for it will get a version without mocks.
 As with `requireInject`, except that the module and its mocks are left in
 the require cache and any future requires will end up using them too. This is
 helpful particularly in the case of things that defer loading.
+
+
+## Brute-force
+
+Since sometimes you need to mock a module which is deep in the hierachy of your require.cache where it is really hard to find out where exactly you required it (It only records where it was required when it was cached) we give you a method to completly delete your require.cache.
```

Comment:
This doesn't actually mention how to use this feature... I'd like to see it documented like `installGlobally` above.